magic-cli

```diff
@@ -43,6 +44,20 @@ class MagicCli
         return description
     end
 
+    def get_parameters(filename)
+      # Optionally, retrieve parameters that are required
+      lines = File.readlines(filename)
+      parameters = lines.map do |line|
```

Comment:
I think we want to stop iterating over the lines after we're past the initial header block. For example, if we had a file like this...

```ruby
#!/usr/bin/env ruby
# Foo bar frobber
# @param foo

def some_method(a)
  # @param some_method_param
end
```

...we wouldn't want to show `some_method_param` in the main output.